testing/gearmand/gearmand.pre-install

```diff
@@ -0,0 +1,15 @@
+#!/bin/sh
+
+## add gearmand group or use exsiting
+if ! getent group gearmand &>/dev/null; then
```

Comment:
To better reiterate @jirutka's point: Never use bashisms in scripts not using bash in shebang (`#!` line) and never use bash for system scripts, because systems, especially systems valueing small size, may not have full blown shell interpreters like bash or zsh out-of-the-box.

You can find boilerplate in wiki:
https://wiki.alpinelinux.org/wiki/APKBUILD_Reference#.24pkgname.pre-install

It would be nice if you could preserve the argument order too, as it helps in mass processing/maintaining them.

(In future AL wants to go away from adduser/addgroup, but we have to live with it for now.)